src/host_capabilities/mod.rs

```diff
@@ -64,7 +85,4 @@ pub enum CallbackRequestType {
         /// Optional - Annotations that must have been provided by all signers when they signed the OCI artifact
         annotations: Option<HashMap<String, String>>,
     },
-
-    /// Lookup the addresses for a given hostname via DNS
-    DNSLookupHost { host: String },
```

Comment:
Considering that @flavio already reviewed this PR, I think we are fine. But I would like to know, does not this removal break the DNS lookup feature? I do not know if this is a real issue. But I would like to bring attention to this removal.  Just in case this change had  unnoticed.

Reply:
CallbackRequestType and DNSLookopHost has been moved to the policy-evaluator, so it should work fine.

Reply:
Yes, it will work fine. The `CallbackRequestType` enum was used only inside of policy-evaluator, but then it started to be used also by the validation guest calls as a way to express their payload.

Now the payload of the guest calls is defined by dedicated types that are defined inside of the SDK. While `CallbackRequestType` has been moved into policy-evaluator, where it should have always belonged

Reply:
Thank you for the explanation! :)